src/main/java/org/jboss/tools/intellij/openshift/actions/project/ChangeActiveProjectAction.java

```diff
@@ -189,4 +190,9 @@ public void run() {
       );
     }
   }
+
+  @Override
+  public @NotNull ActionUpdateThread getActionUpdateThread() {
```

Comment:
This could/should have arguably been fixed in a separate issue. When the cluster isn't reachable, the EDT is stuck waiting and the platform throws an exception indicating this fix.